eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1073,7 +1073,7 @@ protected function assertSubtreeProperties(array $expectedValues, Location $loca
         $repository = $this->getRepository();
         $locationService = $repository->getLocationService();
 
-        if ($location->id === $stopId) {
+        if ($location->id == $stopId) {
```

Comment:
this is probably because you did not cast id's in extractor?
Should rather cast to make them behave same as current code